Python/Product/Analysis/Interpreter/Ast/AstPythonFunction.cs

```diff
@@ -54,6 +54,17 @@ LocationInfo loc
             Locations = loc != null ? new[] { loc } : Array.Empty<LocationInfo>();
         }
 
+        internal AstPythonFunction(IPythonFunction original) {
+            DeclaringModule = original.DeclaringModule;
+            DeclaringType = original.DeclaringType;
+            Name = original.Name;
+            _doc = (original as AstPythonFunction)?._doc ?? original.Documentation;
+            IsClassMethod = original.IsClassMethod;
+            IsStatic = original.IsStatic;
+            _overloads = original.Overloads.ToList();
+            Locations = (original as AstPythonFunction)?.Locations ?? (original as ILocatedMember)?.Locations ?? Array.Empty<LocationInfo>();
```

Comment:
`AstPythonFunction` implements `ILocatedMember`, so no need to check `AstPythonFunction` by itself